course-content/13_working_with_code.qmd

```diff
@@ -131,63 +143,84 @@ When you click on `Help` you see an expanded range of topics:
 Under the topics listed you will find treasure troves of helpful documentation relating to Python. Whether you are relatively new to the language or have years of experience already with it, one of the most helpful sections you will find there is *Python Reference*:
 
 <center>
-    <img src="images/working-with-code/python_documentation.png" size=700 style="margin:auto" border=1/>
+    <img src="images/working-with-code/python_documentation.png" width=95% style="margin:auto" border=1/>
     <p style="text-align: center">
         Official Python Documentation
     </p>
 </center>
 
 Each section provides useful content that is related to the version of Python you are working with! One of the particularly helpful sections for those new to Python is the *Tutorial* section, where you will find tutorials on several of the topics in this course, plus many more.
 
-
+<br>
 *Functions help*  
 
 Functions in Python have excellent documentation that you should be able to access fairly readily. Below are screen shots from the Jupyter Notebook environment that show two distinct ways to access. The example below shows how to get detail about a specific function by calling `help()` function itself and passing in the name of the function of interest:
 
-<left>
-    <img src="images/working-with-code/shiftenter_help.png" size=700 style="margin:auto" border=1/>
-    <p style="text-align: center">
- 
-</left>
+```{python}
+help(round) # Run code as normal to get help information
+```
+
+<br>
+
+
+Within Jupyter environments, it is also possible to place the cursor inside the cell with the function and hit `Shift-tab` to get information:
+
+
+```{python}
+#| error: True
+round() # Shift-tab to get drop-down help in Jupyter 
+```
 
-Within Jupyter Notebook, it is also possible to place the cursor inside the "()" and then pulse the Shift-tab key combination twice to get access to a pop-up:
 
 <left>
-    <img src="images/working-with-code/shifttab_x2.png" size=700 style="margin:auto" border=1/>
+    <img src="images/working-with-code/jl_shifttab.png" size=600 style="margin:auto" border=1/>
```

Comment:
```suggestion
    <img src="images/working-with-code/jl_shifttab.png" size=70% style="margin:auto" border=1/>
```

Changing a few missed plots to % instead of raw pixels.